sphinx/ext/intersphinx/_load.py

```diff
@@ -107,15 +107,19 @@ def fetch_inventory_group(
     app: Sphinx,
     now: int,
 ) -> bool:
-    cache_time = now - app.config.intersphinx_cache_limit * 86400
+    if app.config.intersphinx_cache_limit != -1:
```

Comment:
```suggestion
    if app.config.intersphinx_cache_limit < 0:
```

The docs says *any* negative value would be fine.